src/pretix/control/logdisplay.py

```diff
@@ -421,6 +423,36 @@ def display(self, logentry: LogEntry, data):
         )
 
 
+@log_entry_types.new_from_dict({
+    "pretix.event.order.data_sync.success": _("Ticket data successfully transferred to {provider}."),
```

Comment:
```suggestion
    "pretix.event.order.data_sync.success": _("Data successfully transferred to {provider}."),
```
could be order data as well